### ISSUE TITLE
``dotenv`` can be a dev dependency

### ISSUE THREAD

**Issue by joe-dev-public** [fac24/week4-maria-minju-hussain-hallie#45]
https://github.com/fac24/week4-maria-minju-hussain-hallie/blob/8ed95bead5fdd1a0d7c9b8087fe24140aff833a1/package.json#L25

It's probably not doing any harm, but this package isn't required for deployment, only for local dev testing. So it can be a dev dependency instead of a regular dependency.

There's probably an ``npm`` command, but I think you can just edit your ``package.json`` manually and move this line down into the ``devDependencies`` section.

**Comment by joe-dev-public** [comment]
PS @oliverjam I assume it's also fine to have ``nodemon`` as a dev dependency?

**Comment by oliverjam** [comment]
Yeah anything your server doesn't actually use while it is running (i.e. you don't ever `require` it) should be a devDep. That way your Heroku deploys will be a bit quicker since you won't be installing a bunch of code you don't use
